styleguide.setup.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/no-unassigned-import
+import "@babel/polyfill";
```

Comment:
What's this change doing?

Reply:
I'll add a comment to this file as well, but basically styleguidist doesn't include `regeneratorRuntime` when it builds its files which means that nothing loads when using `async` functions.  The built files should just work in webapp since we're already including `regeneratorRuntime` there.  https://github.com/styleguidist/react-styleguidist/issues/162